exercises/anagram/examples/list-string/package.yaml

```diff
@@ -0,0 +1,17 @@
+name: anagram
```

Comment:
this file is pretty much just "the default package.yaml" - perhaps we should support the absence of this file, in which case the one from `exercises/anagram/package.yaml` is used? the only tricky bit is that it may be overwritten by an earlier example, so we need to back it up beforehand.